### DISCUSSION TITLE
Where is the focusing behaviour?

### DISCUSSION THREAD

**Discussion by AlexCrosby** [jcubic/jquery.terminal#812] [Q&A]
I wan't to change the focus behaviour and I'm looking for where I can find whatever is doing the following:

When you click so that the text area is not focused, when I make a key press the textarea is refocused and the character is inserted. However somehow if I do something live ctrl+c, then it does not refocus the text area. Where is the logic that decides when the textarea should be focued or not and inserts the character?

**Comment by jcubic** [comment]
Sorry, I don't understand. Where exactly is the textarea? Next to Terminal or inside? Can you maybe create a demo that reproduces the issue?

Also, which version of the library you're using?

**Comment by jcubic** [comment]
Is this what you want to do? If yes, can you tell me why?

https://codepen.io/jcubic/pen/xxWPRba

Also, I'm not sure why it doesn't work (the library uses a hidden textarea). But it looks like a bug.

**Comment by jcubic** [comment]
It seems it works, but the `blur` method is jQuery API, Terminal API is `focus(false)`, maybe I should overwrite blur.

https://codepen.io/jcubic/pen/XWEzNKY
